<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSelectWidget</class>
 <widget class="QWidget" name="ImageSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>567</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>资源浏览器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>搜索</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_filter"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="LV_Image_Browse">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QListView" name="LV_Image_selected"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="PB_Confirm">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>LV_Image_Browse</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>ImageSelectWidget</receiver>
   <slot>on_select_item(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LV_Image_Browse</sender>
   <signal>pressed(QModelIndex)</signal>
   <receiver>ImageSelectWidget</receiver>
   <slot>on_selected_image(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PB_Confirm</sender>
   <signal>clicked()</signal>
   <receiver>ImageSelectWidget</receiver>
   <slot>on_pb_confirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>518</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_select_item(QModelIndex)</slot>
  <slot>on_selected_image(QModelIndex)</slot>
  <slot>on_pb_confirm()</slot>
 </slots>
</ui>
